helix-term/src/ui/picker.rs

```diff
@@ -40,7 +40,10 @@ use helix_view::{
 
 use super::{menu::Item, overlay::Overlay};
 
-pub const MIN_AREA_WIDTH_FOR_PREVIEW: u16 = 72;
+pub const MIN_AREA_WIDTH_FOR_PREVIEW: u16 = 40;
+pub const MIN_AREA_HEIGHT_FOR_PREVIEW: u16 = 9;
```

Comment:
9 seems a little low, I don't think we should not below 15 here

Reply:
one of the main goals here is to have it able to show the preview in an 80x24 terminal. would it be better to drop the min width down to 36? (it's worth noting that the interpretation of these constants is different in the pr than previously - the existing code tests against the full width/height, but in this pr it is testing against the width that would be used by just the preview)

Reply:
ah I see I don't have time to test this out at the moment and din\t look at the new code in detail zet so I thaught it had the same meaning as before. I think the old bound was fine so if the definition just changed then its probably ok, I will take a look at that one I have more time.

I just meanr rhar 9 rows for picker and preview would have been a little low (that would have been like 1 or two matches shown at most)